packages/admin-e2e-tests/src/fixtures/plugins.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Internal dependencies
+ */
+import { httpClient } from './http-client';
+
+const wpPluginsEndpoint = '/wp/v2/plugins';
+
+type Plugin = {
+	author: string;
+	name: string;
+	plugin: string;
+	plugin_uri: string;
+	status: 'active' | 'inactive';
+	version: string;
+	description: {
+		raw: string;
+		rendered: string;
+	};
+};
+
+export async function getPlugins(): Promise< Plugin[] > {
+	const response = await httpClient.get( wpPluginsEndpoint );
+	expect( response.statusCode ).toEqual( 200 );
+	return response.data;
+}
+
+export async function deletePlugin( pluginName: string ) {
+	const response = await httpClient.delete(
+		wpPluginsEndpoint + '/' + pluginName
+	);
+	expect( response.statusCode ).toEqual( 200 );
+}
+
+export async function deactivatePlugin( pluginName: string ) {
+	const response = await httpClient.post(
+		wpPluginsEndpoint + '/' + pluginName,
+		{
+			status: 'inactive',
+		}
+	);
+	expect( response.statusCode ).toEqual( 200 );
+}
+
+async function deactivateAndDeletePlugin( pluginName: string ) {
+	await deactivatePlugin( pluginName );
+	await deletePlugin( pluginName );
+}
+export async function deactivateAndDeleteAllPlugins( except: string[] = [] ) {
```

Comment:
Handled this here versus the woocommerce-reset plugin, as there was issues removing multiple plugins during the same call when using the reset plugin -> https://github.com/woocommerce/woocommerce-reset/pull/6#pullrequestreview-810079601